Document-Processing/PDF/Conversions/HTML-To-PDF/NET/Convert-HTML-to-PDF-in-Amazon-ECS-with-Fargate.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Convert HTML to PDF in Amazon ECS with Fargate | Syncfusion
+description: Learn how to convert HTML to PDF in Amazon ECS with Fargate using Syncfusion .NET HTML converter library.
+platform: document-processing
+control: PDF
+documentation: UG
+---
+
+# Convert HTML to PDF file in Amazon ECS with Fargate
+
+The Syncfusion<sup>&reg;</sup> [HTML to PDF converter](https://www.syncfusion.com/document-processing/pdf-framework/net/html-to-pdf) is a .NET library for converting webpages, SVG, MHTML, and HTML to PDF using C#. Using this library, **convert HTML to PDF document using Blink in Amazon ECS with Fargate**.
```

Comment:
https://www.syncfusion.com/document-sdk/net-pdf-library/html-to-pdf - use this latest FT link